google/cloud/ndb/model.py

```diff
@@ -4861,9 +4861,9 @@ def _gql(cls, query_string, *args, **kwargs):
         from google.cloud.ndb import query
 
         return query.gql(
-            "SELECT * FROM {} {}".format(
-                cls._class_name(), query_string, *args, *kwargs
-            )
+            "SELECT * FROM {} {}".format(cls._class_name(), query_string),
+            *args,
+            **kwargs,
```

Comment:
The trailing comma here is not valid in Python 2.7. We might as well remove it now.

Reply:
Hmm, well. Blacken added it. That's kind of a conundrum...

Reply:
I found a way to make blacken happy and get rid of that comma.